### PR TITLE
Setting up writing functions for wav files

### PR DESCRIPTION
**Pull request by rpfisher**
Base io for wave files, and registering io functions for wav files to the TimeSeries class.

### CODE REVIEW

File: gwpy/timeseries/io/wav.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with GWpy.  If not, see <http://www.gnu.org/licenses/>.
+
+"""This module attaches the WAV input output methods to the TimeSeries.
+"""
+
+from ...types.io import wav
```
Comment (duncanmmacleod) [inline gwpy/timeseries/io/wav.py]: @rpfisher does this line work? given that you haven't changed `types.io` I suspect not.

File: gwpy/io/wav.py [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+
+from ..types import TimeSeries
+from .registry import (register_reader, register_writer, register_identifier)
+from .utils import identify_factory
+
+__author__ = "Ryan Fisher <ryan.fisher@ligo.org>"
+
+
+#def read_ascii(filepath, _obj=Series, xcol=0, ycol=1, delimiter=None,
```
Comment (duncanmmacleod) [inline gwpy/io/wav.py]: @rpfisher, why are these comment blocks here?

Comment (rpfisher) [inline gwpy/io/wav.py]: Left those comment blocks in to show the relative difference between wav and another io format.  This was to better enable you and Derek to review it.  We can remove the comments after you and Derek review the code if you prefer.

**Comment by duncanmmacleod** [comment]
@rpfisher, I think it would be cleaner to have all of the new code in the `gwpy.timeseries.io.wav` module, rather than separating into `gwpy.io.wav`, can you combine it all together into a single new module?

**Comment by rpfisher** [comment]
I think that would be less well designed for any future extension of the wav io operations.  Specifically, if we want to make it more broadly usable by other Series subclass objects that aren't themselves subclasses of timeseries.

**Comment by duncanmmacleod** [comment]
I take your point, however, since this is currently restricted to the `TimeSeries` object, I would like it merged. In the future if someone proposes the kind of extensions you suggest, it can be separated appropriately.

**Comment by rpfisher** [comment]
This whole PR is too buggy now that Duncan provided a nice simple testing script.  Closing and starting over!

**Pull request closed without merging**